eng/Versions.props

```diff
@@ -4,6 +4,7 @@
     <VersionPrefix>4.8.0</VersionPrefix>
     <PreReleaseVersionLabel>preview5</PreReleaseVersionLabel>
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <NewtonsoftJsonVersion>12.0.1</NewtonsoftJsonVersion>
```

Comment:
consider moving this to a separate section and annotating it with a comment that this is a static version. Also consider renaming as `NewtonsoftJsonVersionConstant` or some such name .

Reply:
newtonsoft has been removed